solidity/random-beacon/hardhat.config.ts

```diff
@@ -91,6 +92,11 @@ const config: HardhatUserConfig = {
         : undefined,
       tags: ["tenderly"],
     },
+    development: {
+      url: "http://localhost:8545",
+      chainId: 1101,
+      tags: ["local"],
```

Comment:
Note that the `local` tag will change to `allowStubs` in https://github.com/keep-network/keep-core/pull/3052.